chroma_core/models/stratagem.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2019 DDN. All rights reserved.
+# Use of this source code is governed by a MIT-style
+# license that can be found in the LICENSE file.
+
+from django.db import models
+
+from chroma_core.lib.cache import ObjectCache
+from chroma_core.lib.job import Step
+from chroma_core.models import Job
+from chroma_core.models import StateChangeJob
+from chroma_help.help import help_text
+
+
+class Stratagem(models.Model):
+    id = models.IntegerField(primary_key=True, default=1)
+    interval = models.IntegerField(help_text="Interval value in which a stratagem run will execute")
```

Comment:
What unit is the interval and durations in?

Reply:
I was thinking a unit of days would make the most sense. In addition to that we would probably need to specify a time of day (in UTC I would imagine).

Reply:
We'd probably want it to have a more granular lower bound. We can always scale up to days, but can't scale down from them.

Reply:
Do you guys have an opinion on adding a units dropdown?

Reply:
I'd be fine with using a more configurable picker for the GUI. 

On the backend, we want to store in granular units, even if we end up using coarser units on the front-end.

Reply:
yes, absolutely. I'll check into what systemd uses for timers but I was thinking of using minutes.

Reply:
Looks like you can specify units for systemd timers (eg. 15min). I'm starting to think seconds might be a good unit for the backend. What do you guys think?

Reply:
Seconds should be more than granular enough.